package.json

```diff
@@ -51,63 +51,63 @@
   "license": "MIT",
```

Comment:
@jaredrethman something in here is breaking our CSS processing, can you investigate? You can test by adding:

```
body {
	background: var(--c-black);
	color: var(--c-white);

	@media (--bp-small) {
		background: var(--c-white);
		color: var(--c-black);
	}

}
```

into `/assets/css/base/index.css` and noting that the media queries don't work after npm install on this branch, but they work on master

Reply:
@timwright12 & @jaredrethman. It's because `@custom-media` rules are inside `:root` which they shouldn't.

I think they working before with luck:)

So to fix this `media-queries.css` should look like this without `:root`.

```css
/*
 * Media Queries
 */
@custom-media --bp-tiny ( min-width: 25em ); /* 400px */
@custom-media --bp-small ( min-width: 30em ); /* 480px */
@custom-media --bp-medium ( min-width: 48em ); /* 768px */
@custom-media --bp-large ( min-width: 64em ); /* 1024px */
@custom-media --bp-xlarge ( min-width: 80em ); /* 1280px */
@custom-media --bp-xxlarge ( min-width: 90em ); /* 1440px */

/* WP Core Breakpoints (used for the admin bar for example) */
@custom-media --wp-small ( min-width: 600px );
@custom-media --wp-medium-max (max-width: 782px);
```

Jared, let me know if you have questions or need help.

Reply:
@samikeijonen @timwright12 haven't had much time lately to fix/address this. The small investigations I did recently pointed to `post-css-env` as the culprit. Quick fix was to downgrade it back, but if you're suggestion works - that would be 1000% better.

Can look into this over the coming weekend, unless you're able to tackle it sooner :)

Reply:
@samikeijonen your suggestion worked perfectly. @timwright12 let me know if you have success with the latest commit - https://github.com/10up/theme-scaffold/pull/162/commits/87f2abe906aad56c614c7debb4501b8658743ed9.